src/request/create/CreateHigh.js

```diff
@@ -0,0 +1,114 @@
+/* global RecoveryWords */
+/* global ValidateWords */
+/* global ChooseIdenticon */
+/* global PassphraseInput */
+/* global Key */
+/* global KeyStore */
+
+class CreateHigh {
+    /**
+     * @param {CreateRequest} request
+     * @param {Function} resolve
+     * @param {Function} reject
+     */
+    constructor(request, resolve, reject) {
+        this._resolve = resolve;
+        this._reject = reject;
+
+        // set html elements
+        /** @type {HTMLDivElement} */
+        this.$chooseIdenticon = (document.getElementById(CreateHigh.Pages.CHOOSE_IDENTICON));
```

Comment:
surrounding parenthesis are not necessary

Reply:
Yes they are, as otherwise TypeScript does not apply the @type annotation. Even when it's two lines like this.